frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts

```diff
@@ -1,6 +1,6 @@
 import { browser, $, ExpectedConditions as until, by, element } from 'protractor';
 
-export const operatorModal = $('.pf-c-modal-box');
+export const operatorModal = $('.co-catalog-page__overlay');
```

Comment:
This change is necessary as the above modal element is using pattern fly class which is quite unreliable and tests are failing because of it.

Reply:
I wasn't directly inquiring if this was a needed change - I was inquiring if it will impact other tests. I was also mentioning because of this change, I am unable to give an `approved` label... so the best I can give is `lgtm` - the same as Len.